### PR TITLE
feat: Enable Holodex, misc: Options Page

### PR DESCRIPTION
**Pull request by XangelMusic**
Created a setting to enable LiveTL on Holodex in Advanced > Additional sites in Options.

Set open_in_tab to false in manifest for Options Page (Firefox) and Embedded Options (Chrome).

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "LiveTL - Translation Filter for Streams",
   "options_ui": {
     "page": "options.html",
-    "open_in_tab": true
+    "open_in_tab": false
```
Comment (r2dev2) [inline src/manifest.json:5]: does this need to be set to false? would not want to change existing behaviour

File: src/ts/content_scripts/injector.ts [hunk 2]
```diff
@@ -10,6 +10,27 @@ for (const eventName of ['visibilitychange', 'webkitvisibilitychange', 'blur'])
...
+    if (hostname === 'holodex.net' && !holodexEnabled.get()) { return; }
+  } catch (e) {
+    if (e instanceof Error) {
+      holodexEnabled.subscribe($holodexEnabled => {
+        console.log( document.referrer );
+        console.log({ $holodexEnabled });
+      });
+      console.debug('Could not get hostname', { e });
+    } else {
+      throw e;
```
Comment (r2dev2) [inline src/ts/content_scripts/injector.ts]: Can you remove the `throw e`? I would not want the function to crash if something goes wrong.

Also, can you replace the debugging of checking `holodexEnabled` with that singular `console.debug`?

**Review by r2dev2 — CHANGES_REQUESTED** [review]
Sorry for taking so long to review this, was a bit busy with university. I have put a few pieces of feedback.

Also, this week, I'm going to be working on the mv3 re-write so I will be a lot more active on GitHub.

File: src/manifest.json (hunk 1 above)
Comment (XangelMusic) [reply src/manifest.json:5]: This allows an Options Tab in the Manage Extensions page in Firefox, or just gives a simple pop-up in Chrome.

Its just more cleaner imo, and doesn't change the overall behavior of the extension.

Comment (r2dev2) [reply src/manifest.json:5]: Oh wow I actually did not know that is what that does. That's great! Thanks for adding this.

**Review by r2dev2 — APPROVED** [review]

**Pull request merged**